### PR TITLE
add missing 'mc admin inspect' examples

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/admin-inspect.go [hunk 1]
```diff
@@ -62,7 +62,12 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
+EXAMPLES:
+  1. Download 'xl.meta' for a specific object from all the drives in a zip file.
+     {{.Prompt}} {{.HelpName}} myminio/bucket/test*/xl.meta
 
+  2. Download all constituent parts for a specific object, and optionally encrypt the downloaded zip.
```
Comment (vadmeste) [inline cmd/admin-inspect.go:69]: I believe a third example is needed with `**` to recursively download a folder from all disks

`mc admin inspect myminio/.minio.sys/**`

**Review by vadmeste — COMMENTED** [review]
One comment

File: cmd/main.go [hunk 2]
```diff
@@ -299,12 +300,7 @@ func initMC() {
 	loadRootCAs()
 }
 
-func installAutoCompletion() {
-	if runtime.GOOS == "windows" {
-		console.Infoln("autocompletion feature is not available for this operating system")
-		return
-	}
-
+func getShellNave() (string, bool) {
```
Comment (vadmeste) [inline cmd/main.go:303]: s/Nave/Name

File: cmd/main.go [hunk 3]
```diff
@@ -315,19 +311,26 @@ func installAutoCompletion() {
 				"no SHELL environment variable found")
 		}
 		shellName = strings.TrimSpace(string(ppName))
+		return strings.ToLower(filepath.Base(shellName)), false
```
Comment (vadmeste) [inline cmd/main.go:314]: This is duplicated - check L316

File: cmd/admin-inspect.go [hunk 4]
```diff
@@ -97,7 +106,15 @@ func mainAdminInspect(ctx *cli.Context) error {
 	splits := splitStr(aliasedURL, "/", 3)
 	bucket, prefix := splits[1], splits[2]
 
-	key, r, ierr := client.Inspect(context.Background(), madmin.InspectOptions{Volume: bucket, File: prefix})
+	shellName, _ := getShellNave()
+	if shellName != "bash" && strings.Contains(prefix, "*") {
+		console.Infoln("Your shell is auto determined as '" + shellName + "', wildcard patterns are only supported with 'bash' SHELL.")
```
Comment (vadmeste) [inline cmd/admin-inspect.go:111]: /cc @klauspost

**Review by vadmeste — COMMENTED** [review]
Few coments

File: cmd/admin-inspect.go (hunk 4 above)
Comment (klauspost) [reply cmd/admin-inspect.go:111]: It works fine on Windows. @harshavardhana Why is this being diabled?

Comment (harshavardhana) [reply cmd/admin-inspect.go:111]: on zsh and tcsh shells `**` is escaped by the shell and it doesn't work.

Comment (klauspost) [reply cmd/admin-inspect.go:111]: Then exclude those explicitly or have an OS check.

Comment (harshavardhana) [reply cmd/admin-inspect.go:112]: fixed @klauspost

**Comment by harshavardhana** [comment]
PTAL @vadmeste @klauspost

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**